Sources/LiveViewNative/Coordinators/LiveSessionConfiguration.swift

```diff
@@ -15,6 +15,11 @@ public struct LiveSessionConfiguration {
     ///
     /// By default, no connection params are provided.
     public var connectParams: ((URL) -> [String: Any])? = nil
+  
+    /// Optional headers that are passed along with estabilishing the initial connection.
+    ///
+    /// By default, no addtional headers are provided.
+    public var headers: [String: String] = [:]
```

Comment:
Can we make this optional with a default value of `nil`? This would match the `URLRequest` API, and makes it more clear that the headers are optional.

Reply:
Done.